Essentials/src/messages.properties

```
@@ -484,6 +485,7 @@ seenOnline=\u00a76Player\u00a7c {0} \u00a76has been \u00a7aonline\u00a76 since \
 sellBulkPermission=\u00a76You do not have permission to bulk sell.
 sellHandPermission=\u00a76You do not have permission to hand sell.
 serverFull=Server is full\!
+serverReloading=There's a good chance you're probably reloading your server right now. If that's the case, why do you hate yourself? Except no support from the Essentials team when using /reload
```

```suggestion
serverReloading=There's a good chance you're probably reloading your server right now. If that's the case, why do you hate yourself? Expect no support from the EssentialsX team when using /reload.
```
Others might have other suggestions on wording.

---

https://ptb.discordapp.com/channels/390942438061113344/395279435538104320/709404997560959077